supervisor/store/__init__.py

```diff
@@ -63,11 +63,14 @@ def get(self, slug: str) -> Repository:
         return self.repositories[slug]
 
     async def load(self) -> None:
-        """Start up add-on management."""
-        # Init custom repositories and load add-ons
-        await self.update_repositories(
-            self._data[ATTR_REPOSITORIES], issue_on_error=True
-        )
+        """Start up add-on store management."""
+        # Make sure the built-in repositories are all present
+        # This is especially important when adding new built-in repositories
+        # to make sure existing installations have them.
+        all_repositories: set[str] = set(self._data.get(ATTR_REPOSITORIES, [])) | {
+            repo.value for repo in BuiltinRepository
+        }
```

Comment:
Since we should make a copy of `DEFAULT_REPOSITORIES` in order to use it in the validate, maybe better to just define it as a `set` to begin with? Then we can use it here as is and in the validate we can do `list(DEFAULT_REPOSITORIES)` (which also conveniently makes a copy).

Reply:
Yeah seems sensible :+1: